FluentTerminal.App/Strings/de/Resources.resw

```diff
@@ -748,7 +748,7 @@ Fuzzy</comment>
     <value>Fenster ein/ausblenden</value>
     <comment>Command</comment>
   </data>
-  <data name="NewSshTab.Text" xml:space="preserve">
+  <data name="NewRemoteTab.Text" xml:space="preserve">
```

Comment:
this has been renamed to NewSshTab with #410

Reply:
There's _New remote tab_ in the main menu, and the control name is `NewRemoteTab`. I don't see `NewSshTab` anywhere except in `Command` enumeration, but in this case `Text` property doesn't make any sense.
Honestly, I can't remember now if I've renamed the control to `NewRemoteTab`, but resources are coming from POEditor anyway...

Reply:
Fixed.